orochi/client.py

```diff
@@ -685,6 +704,15 @@ def do_status(self, s=''):
             parts.append('(%s)' % track_year)
         print(' '.join(parts) + '.')
 
+        # Log the current song to a file (can be used by other programs)
+        if self._log_current_song is True:
+            cachedir = get_orochi_xdg_dir('XDG_CACHE_HOME', '.cache')
+            filename = os.path.join(cachedir, 'current_song.txt')
+            track_attributes = (track_name, track_performer, track_album, track_year)
+            track_attributes = ['' if v is None else unicode(v) for v in track_attributes]
```

Comment:
apparantly not, I tried str(v)  [ since there are numberics that were coming in attributes ] ,   which failed the conversion because of unicode issues.

uncode(v) converts strings as well as numbers.